spec/text_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper"
+
+describe "PDF::Core::Text" do
+  before do
+    class TextMock
+      include PDF::Core::Text
+
+      attr_reader :text
+
+      def add_content(str)
+        @text ||= ""
+        @text << str
+      end
+    end
+    @mock = TextMock.new
+  end
+  describe "horizontal_text_scaling" do
+    describe "called without argument" do
+      before do
+        @result = @mock.horizontal_text_scaling
```

Comment:
This line can be moved into the next example. Generally it's recommended to use `let` syntax or straight inline setup instead of instance variables.